scripts/build_osm_network.py

```diff
@@ -753,7 +882,7 @@ def built_network(inputs, outputs, geo_crs, distance_crs):
         logger.info(
             f"Stage 4/5: Aggregate close substations: enabled with tolerance {tol} m"
         )
-        lines, buses = merge_stations_lines_by_station_id_and_voltage(
+        lines, links, buses = merge_stations_lines_by_station_id_and_voltage(
```

Comment:
There is no need to have the links processing and output into merge_stations_lines_by_station_id_and_voltage, I'd suggest to remove that

Reply:
Yes, I see: `merge_stations_lines_by_station_id_and_voltage( )` is executed only under a condition and may by bypassed. Moved a lines/links split into a higher-level function.